components/tcf/firstLayer/src/index.js

```diff
@@ -1,114 +1,204 @@
+    <>
+      <SuiButton onClick={handleSettingsClick} design="outline">
+        {i18n.CONFIGURE_BUTTON}
+      </SuiButton>
+      <SuiButton onClick={handleSaveExitClick}>
+        {i18n.CONTINUE_NAVIGATION_BUTTON}
+      </SuiButton>
+    </>
+  )
+
   return (
-    <div className="sui-TcfFirstLayer">
-      <SuiModal
-        isOpen={modalOpen}
-        closeOnOutsideClick
-        closeOnEscKeyDown
-        header={<strong>TcfFirstLayer</strong>}
-        iconClose={<IconClose />}
-        onClose={handleCloseModal}
-      >
-        {state && vendorListState && (
-          <InfoCard
-            name="purposes"
-            title="We care about your privacy"
-            descriptions={vendorListState.purposes}
-            state={state.purposes}
-            onStateChange={handlePurposesChange}
-          />
-        )}
-        <br />
-        <br />
-        <SuiButton onClick={handleSettingsClick}>Manage Settings</SuiButton>
-        <SuiButton onClick={handleSaveExitClick} disabled={saveButtonDisabled}>
-          Save and Exit
-        </SuiButton>
-        <SuiButton onClick={handleAcceptAll}>Accept All</SuiButton>
-      </SuiModal>
+    <div className={CLASS}>
+      {isMobile && (
+        <SuiModal
+          isOpen={show}
+          closeOnOutsideClick
+          closeOnEscKeyDown
+          header={<img className={`${CLASS}-logo`} src={logo} alt="logo" />}
+          iconClose={<IconClose class={`${CLASS}-icon-close`} />}
+          onClose={handleCloseModal}
+          fitContent
+        >
+          <div className={`${CLASS}-body`}>
+            <span
+              className={`${CLASS}-body-info`}
+              ref={textRef}
+              dangerouslySetInnerHTML={{__html: i18n.BODY}}
+            />
+            <div className={`${CLASS}-buttons`}>
+              <Buttons />
+            </div>
+          </div>
+        </SuiModal>
+      )}
+      {!isMobile && (
+        <div className={`${CLASS}-notification`}>
+          <SuiNotification
+            position="bottom"
+            autoClose="manual"
+            show={show}
+            showCloseButton={false}
+            variation="positive"
+            type="system"
+          >
+            <div
+              className={
+                isTablet
+                  ? `${CLASS}-body ${CLASS}-body ${CLASS}-body--footer ${CLASS}-body--tablet`
+                  : `${CLASS}-body ${CLASS}-body--footer`
+              }
+            >
+              <span
+                className={`${CLASS}-body-info`}
+                ref={textRef}
+                dangerouslySetInnerHTML={{__html: i18n.BODY}}
+              />
+              <div
+                className={
+                  isTablet
+                    ? `${CLASS}-buttons ${CLASS}-buttons--footer ${CLASS}-buttons--tablet`
+                    : `${CLASS}-buttons ${CLASS}-buttons--footer`
+                }
+              >
+                <Buttons />
+              </div>
+            </div>
```

Comment:
IMO, this is a duplication that may be extracted to a component which accepts the `className` variations

Reply:
There are only two divs, I don't think that we need an extracted component to create a div with some classes. Do you think some function as classNameGeneration would be better? @alextremp

Reply:
No no, I refer to this: both inner HTML structure are the same

```
          <div className={`${CLASS}-body`}>
            <span
              className={`${CLASS}-body-info`}
              ref={textRef}
              dangerouslySetInnerHTML={{__html: i18n.BODY}}
            />
            <div className={`${CLASS}-buttons`}>
              <Buttons />
            </div>
          </div>
```

This has className variations depending on the device (desktop, mobile, tablet) but essentially, it's a structure that can be extracted to a component.

Reply:
done, thanks